_posts/2015-03-27-ApiBlueprint-button.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Sharing is caring - socialize you're public API blueprint
+excerpt: Learn the art of API blueprint through socialization and collaboration.
+layout: post
+date: 2015-03-27 00:00:00 +0100
+author: vincenzo
+published: true
+commets: true
+---
+
+We have recently received a lot of feedback from users that would like to learn more about API Blueprint and how to present stunning and beautiful documentation.
```

Comment:
'Have you ever stumbled upon an API that was so beautifully documented and wondered, how did they do that? Well, at Apiary we believe that effective learning happens through socialization and collaboration. To help support this belief and the inspiration that stems from beautifully documented APIs, we added the ability to socialize blueprints.'

kind of bland, but please, feel free to modify